mscore/events.cpp

```diff
@@ -368,6 +404,26 @@ void ScoreView::mousePressEventNormal(QMouseEvent* ev)
 
 void ScoreView::mousePressEvent(QMouseEvent* ev)
       {
+
+      if (tripleClickPending) {
+            qDebug()<<"triple click detected...";
```

Comment:
Could you please remove these `qDebug()` messages? They can be easily added if needed, and it is better not to keep them in the common codebase in most cases.

Reply:
Thanks. Hadn't spotted that I'd left those in there.